Doc fixes (#1390)
dfinity/portal: docs/references/glossary.md

account is a ledger account whose fiduciary is a non-canister
[principal](#principal).

#### canister development kit (CDK)

> A **Canister Development Kit** is an adapter used by the IC SDK so a programming language has the features needed to create and manage canisters. The IC SDK comes with a few CDKs already installed for you so you can use them in the language of yoru choice. 

Reviewer: ```suggestion
A **Canister Development Kit** is an adapter used by the IC SDK that provides a programming language with the features necessary to create and manage canisters. The IC SDK comes with a few CDKs already installed for you so you can use them in the language of your choice. 
```
